tests/util/base.py

```diff
@@ -37,6 +38,22 @@ def dbloader(config):
     setup_test_db()
 
 
+@yield_fixture
+def patched_gevent_sleep(monkeypatch):
+    monkeypatch.setattr('gevent.sleep', mock.Mock())
+    yield
+    monkeypatch.undo()
+
+
+@yield_fixture
+def patched_gevent(monkeypatch):
+    monkeypatch.setattr('gevent.sleep', mock.Mock())
+    monkeypatch.setattr('gevent.spawn', mock.Mock())
```

Comment:
OK. How do you think we should denote what functionality a test is supposed to be testing? Would be great to check with @bengotow and do something consistent with how we're doing it in N1.